packages/core/strapi/lib/types/core/config/server.d.ts

```diff
@@ -0,0 +1,37 @@
+import type { ConfigExport } from './shared.d.ts';
+
+export type AppProp = {
+  keys: string[];
+};
+
+export type CronProp = {
```

Comment:
Since we export everything at the same level, it means that we'll be able to do Config.CronProps, won't that be a bit strange since we won't know those props are related to the server config? (same for other types)